Update moloch_update_get.sh

sickcodes opened the pull request:
Update critical environmental variables.

Reduce `if $? == 0` to `&&`

**Clearly describe the problem and solution**
Escapable environmental variables.

**Relevant issue number(s) if applicable**
#976661

**Did you run `npm run lint` from the viewer or parliament directory (whichever you are making changes to) and correct any errors**
Shell code

**Did you Ensure that all tests still pass by navigating to the `tests` directory and running `./tests.pl --viewer`**
No

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

awick commented:
Sorry at this point I'm not going to accept a patch that changes directories used without more information.

sickcodes commented:
I can change it back to /tmp but that folder is publicly writable.

Previously, any non-privileged user could create a file called oui.txt or ipv4-address-space.csv and the script will fail.

sickcodes commented:
/opt is only readable and writeable by root

sickcodes commented:
Secondly, I changed cp to mv so the file is removed after use.

awick commented:
So the reason /tmp was used and the files were left around was so -N could be used. But since we no longer download the large geo files this isn't a big deal. If you don't want to write in /tmp then instead of writing in the random /top you could just write in the final location with a .tmp suffix before doing the move.

sickcodes commented:
I removed the paths altogether and added the .tmp suffix to the downloads.
I also added the force tag, `mv -f`, to ensure that the temporary file always goes over the old one, with zero "downtime"

sickcodes commented:
Cool with current working directory paths or do you still want final directory?

Would become pretty unreadable but I do it if you reckon it's a better place to download to.

awick commented on release/moloch_update_geo.sh:
```diff
@@ -2,25 +2,23 @@
 DEST_DIR="${MOLOCH_DIR:-BUILD_MOLOCH_INSTALL_DIR}/etc"
-TIMEOUT=${WGET_TIMEOUT:-30}
+TIMEOUT="${WGET_TIMEOUT:-30}"
 
```
Just add a cd $DEST_DIR if you don't want to use $DEST_DIR/... everywhere

sickcodes replied:
Resolved :)

awick commented on release/moloch_update_geo.sh:
```diff
@@ -2,25 +2,25 @@
 
-# Work on temp dir to not affect current working files
-cd /tmp
+cd "{$DEST_DIR}"
```
i think it needs to be ${DEST_DIR}

sickcodes replied:
Embarrassing typo resolved.

sickcodes replied:
Working on ubuntu-16 fix

sickcodes commented:
A bit more going on that expected, working on it.

sickcodes commented:
Closed, in favour of other PR https://github.com/aol/moloch/pull/1518
